login/UserData.js

```diff
@@ -0,0 +1,11 @@
+class UserData{
+    constructor(roleId,login,password,name,surname,birthday){
+        this.login=login;
+        this.password=password;
+        this.name=name;
+        this.surname=surname;
+        this.roleId=roleId;
+        this.birthday=birthday;
```

Comment:
I'm sure I've asked about this before, sorry for forgetting, what is the purpose of the birthday field on the user?

Reply:
it will be info (like name and surname) what user can change,we want to add more functional for practice